exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestInfoSchema.java

```diff
@@ -183,6 +183,16 @@ public void describeTable() throws Exception{
         .go();
   }
 
+  @Test
+  public void describeTableWithTableKeyword() throws Exception {
+    testBuilder()
+        .sqlQuery("DESCRIBE TABLE CATALOGS")
+        .unOrdered()
+        .optionSettingQueriesForTestQuery("USE INFORMATION_SCHEMA")
```

Comment:
This applies for both sqlQuery and baselineQuery? Correct?

Reply:
right, will add same option for the baseline query.

Reply:
Use can add before calling testBuilder: `test("USE INFORMATION_SCHEMA")`

Reply:
gotcha